server/views/resources/shop.jade

```diff
@@ -29,15 +33,17 @@ block content
     hr
     .row
         .col-xs-12.col-sm-10.col-sm-offset-1.col-md-8.col-md-offset-2.text-center
-            img.img-responsive.img-center(src='//i.imgur.com/BTtb8P0g.jpg')
+            a(href="//m.do.co/c/ba8a71773373" target='_blank' onClick="ga('send', 'event', 'Shop', 'clicked', '$10 Digital Ocean credit');")
+              img.img-responsive.img-center(src='//i.imgur.com/BTtb8P0g.jpg')
             h3 Get $10 in credit for your own Digital Ocean server
             p Digital Ocean gives our community $25 in credit after you spend $25.
             a.btn.btn-cta.signup-btn.btn-block(href="//m.do.co/c/ba8a71773373" target='_blank' onClick="ga('send', 'event', 'Shop', 'clicked', '$10 Digital Ocean credit');") Claim your $10 credit
 
     hr
     .row
         .col-xs-12.col-sm-10.col-sm-offset-1.col-md-8.col-md-offset-2.text-center
-            img.img-responsive.img-center(src='http://i.imgur.com/YqZqTN6g.jpg')
+            a(href="//www.namecheap.com/?aff=102130" onClick="ga('send', 'event', 'Shop', 'clicked', 'Namecheap domain');")
```

Comment:
Expect the name cheap link. It was discussed in the issue. Namecheap affiliate link does not work if it opens in a new tab.